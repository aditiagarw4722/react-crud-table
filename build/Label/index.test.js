"use strict";

var _react = _interopRequireDefault(require("react"));

var _shallow = _interopRequireDefault(require("react-test-renderer/shallow"));

var _ = _interopRequireDefault(require("."));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

describe('Label', function () {
  it('renders properly', function () {
    var renderer = new _shallow["default"]();
    var result = renderer.render( /*#__PURE__*/_react["default"].createElement(_["default"], null, "Test"));
    expect(result).toMatchSnapshot();
  });
});