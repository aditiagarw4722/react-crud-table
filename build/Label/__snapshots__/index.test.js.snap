// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label renders properly 1`] = `
<Label>
  Test
</Label>
`;
